tests/baselines/reference/commentsOnJSXExpressionsArePreserved(jsx=preserve,module=commonjs).js

```diff
@@ -0,0 +1,27 @@
+//// [commentsOnJSXExpressionsArePreserved.tsx]
+// file is interntionally not a module - this tests for a crash in the module/system transforms alongside the `react-jsx` and `react-jsxdev` outputs
+namespace JSX {
+    interface IntrsinsicElements { [index: string]: any }
```

Comment:
is this supposed to be named `IntrinsicElements`?

Reply:
Yep, but I only added it to ensure there wasn't error noise for things I wasn't testing - guess I don't need intrinsic elements for that - an empty JSX namespace will do.